src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -970,15 +974,25 @@ public void onSessionAccept( JingleSession peerJingleSession,
         // Extract and store various session information in the Participant
         participant.setRTPDescription(answer);
         participant.addTransportFromJingle(answer);
-        participant.addSSRCsFromContent(answer);
-        participant.addSSRCGroupsFromContent(answer);
+
+        try
+        {
+            participant.addSSRCsAndGroupsFromContent(answer);
```

Comment:
Where can I read some more about that ? Seems to be confusing to me. When I see RTP encodings I think of RTP payloads description.